resources/public/api/conf/4.0/coding_out_underpayments_create_or_amend.yaml

```diff
@@ -38,7 +38,7 @@ requestBody:
             $ref: './examples/createOrAmendCodingOutUnderpayments/def1/request.json'
 
 responses:
-  "200":
+  "204":
     description: Success
```

Comment:
`Success(No Content)`
Needs to be done for [opt out](https://github.com/hmrc/self-assessment-accounts-api/blob/fafaca58b20d6d8cf81a18706ae7c249a4bea516/resources/public/api/conf/4.0/opt_out_of_coding_out.yaml#L34 ) & [opt in](https://github.com/hmrc/self-assessment-accounts-api/blob/fafaca58b20d6d8cf81a18706ae7c249a4bea516/resources/public/api/conf/4.0/opt_in_to_coding_out.yaml#L34)